packages/validation-manager/src/BundlerCollectorTracer.ts

```diff
@@ -220,7 +220,7 @@ export function bundlerCollectorTracer (): BundlerCollectorTracer {
           const addr = toAddress(log.stack.peek(1).toString(16))
           const topLevelTargetAddress = toHex(addr)
           // stack.peek(2) - value
-          const ofs = parseInt(log.stack.peek(3).toString())
+          // const ofs = parseInt(log.stack.peek(3).toString())
           // stack.peek(4) - len
           // const topLevelMethodSig = toHex(log.memory.slice(ofs, ofs + 4))
 
```

Comment:
this is destructive: why break 4337 ? 
you can ignore this data if you will in the parser, instead of making the tracer more incompatible

Reply:
It is not more or less incompatible than before. It is just that `topLevelMethodSig` does not fit the system as we decided not to have a specific Factory method.  Note that it seems to be the case for ERC-4337 as well, the `createSender(bytes)` method is absolutely not a part of the ERC and the tracer used it as an indicator of a factory frame. This is not correct.

Reply:
the entire tracer is implementation-specific. it is not part of the ERC-4337, but bound to the EntryPoint and 
EntryPointSimulation  implementations.
We do want to remove it, once we switch to calltracer-based tracer, but we wanted to do it in a separate phase, not as part of the 7560 impl.